### PR TITLE
Document changes in `kubectl-gs` commands towards cluster name, node pool name

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/18247

### CODE REVIEW

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ Example command for an AWS cluster:
 
 ```nohighlight
 kubectl gs template cluster \
-  --provider aws
-  --master-az eu-central-1a \
+  --provider aws \
+  --control-plane-az eu-central-1a \
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/template-cluster.md:74]: Boyscouting. This change has been made some time ago.

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 2]
```diff
@@ -70,15 +70,15 @@ Example command for an AWS cluster:
...
   --external-snat true \
-  --name "Cluster #2" \
+  --description "Cluster #2" \
   --pods-cidr 10.2.0.0/16 \
   --owner acme \
-  --release 11.2.1 \
+  --release 15.0.0 \
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/template-cluster.md:79]: Also updating to a current release.

File: src/content/ui-api/kubectl-gs/template-cluster.md [hunk 3]
```diff
@@ -173,18 +175,19 @@ status: {}
...
-    release.giantswarm.io/version: 11.2.1
+    release.giantswarm.io/version: 15.0.0
   name: osss7
   namespace: default
 spec:
   availabilityZones:
   - eu-central-1a
   instanceType: m5.xlarge
-status: {}
 ```
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/template-cluster.md:193]: Boyscouting: updated the output with what the current command produces.

File: src/content/ui-api/kubectl-gs/template-nodepool.md [hunk 4]
```diff
@@ -128,6 +136,13 @@ spec:
...
+    instanceDistribution:
+      onDemandBaseCapacity: 0
+      onDemandPercentageAboveBaseCapacity: 100
     worker:
-      instanceType: m5.xlarge
+      instanceType: m5.4xlarge
+      useAlikeInstanceTypes: false
+status:
+  provider:
+    worker: {}
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/template-nodepool.md:147]: Boyscouting. Updated the output with the current command's result.

**Review by axbarsan — APPROVED** [review]

**Review by pipo02mix — APPROVED** [review]
Please inform properly customers as some of them have automation generating templates with kgs

**Comment by marians** [comment]
> Please inform properly customers as some of them have automation generating templates with kgs

Yes, currently writing an extensive change log message.

**Pull request merged**